src/components/explorer/ExplorerIntroState.js

```diff
@@ -25,6 +25,17 @@ const ExplorerIntroState = ({ crime, place, participation, until }) => {
   const isArson = crime === 'arson'
   const { nibrs, srs } = ucrParticipationLookup(place)
   const untilUcr = participation.find(p => p.year === until)
+
+  let reportYrStr = ''
+  if (nibrs) {
+    reportYrStr = (
+      <span>
+        {lookupUsa(place).display} State UCR Program began submitting{' '}
```

Comment:
I don't believe "State" should be capitalized here.

What do you think about amending this sentence to have a bit more context, something like:
> The {state} UCR Program has historically submitted {SRS} data and began including {NIBRS} data reports in 2012.

Reply:
That was just feedback from the verbiage the PO wanted so I mirrored it. But I agree

Reply:
Cool, I think we should make this suggested change since we believe it is clearer for users. We will make sure to test this in the next user testing session!